src/main/java/teamroots/embers/register/BlockRegister.java

```diff
@@ -2,184 +2,373 @@
+    public static final Block ASHEN_STONE_SLAB_DOUBLE = new BlockDoubleSlabBase(Material.WOOD, "ashen_stone_slab_double", false, ASHEN_STONE_SLAB).setHarvestProperties("pickaxe", 0).setHardness(1.6f).setLightOpacity(16);
+    public static final Block ASHEN_TILE = new BlockBase(Material.ROCK, "ashen_tile", true).setHarvestProperties("pickaxe", 0).setHardness(1.6f).setLightOpacity(16);
+    public static final Block ASHEN_TILE_SLAB = new BlockSlabBase(ASHEN_TILE_SLAB_DOUBLE, "ashen_tile_slab", true).setHarvestProperties("pickaxe", 0).setIsFullCube(false).setIsOpaqueCube(false).setHardness(1.6f).setLightOpacity(1);
+    public static final Block ASHEN_TILE_SLAB_DOUBLE = new BlockDoubleSlabBase(Material.WOOD, "ashen_tile_slab_double", false, ASHEN_TILE_SLAB).setHarvestProperties("pickaxe", 0).setHardness(1.6f).setLightOpacity(16);
+    public static final Block AUTO_HAMMER = new BlockAutoHammer(Material.ROCK, "auto_hammer", true).setHarvestProperties("pickaxe", 0).setIsFullCube(false).setIsOpaqueCube(false).setHardness(1.6f).setLightOpacity(0);
+    public static final Block BEAM_CANNON = new BlockBeamCannon(Material.ROCK, "beam_cannon", true).setIsFullCube(false).setIsOpaqueCube(false).setHarvestProperties("pickaxe", 0).setHardness(1.6f);
+    public static final Block BEAM_SPLITTER = new BlockBeamSplitter(Material.ROCK, "beam_splitter", true).setIsFullCube(false).setIsOpaqueCube(false).setHarvestProperties("pickaxe", 0).setHardness(1.6f);
+    public static final Block BIN = new BlockBin(Material.ROCK, "bin", true).setIsFullCube(false).setIsOpaqueCube(false).setHarvestProperties("pickaxe", 0).setHardness(1.6f);
+    public static final Block BLOCK_ALCHEMICAL_REDSTONE = new BlockMolten(fluid_alchemical_redstone);
+    public static final Block BLOCK_ALUMINUM = new BlockBase(Material.ROCK, "block_aluminum", true).setBeaconBase(true).setHarvestProperties("pickaxe", 1).setHardness(1.6f).setLightOpacity(16).setCreativeTab(Embers.resource_tab);
+    public static final Block BLOCK_ASTRALITE = new BlockBase(Material.ROCK, "block_astralite", true).setHarvestProperties("pickaxe", 2).setHardness(1.6f).setLightOpacity(16).setCreativeTab(Embers.resource_tab).setCreativeTab(Embers.resource_tab);
+    public static final Block BLOCK_BRONZE = new BlockBase(Material.ROCK, "block_bronze", true).setBeaconBase(true).setHarvestProperties("pickaxe", 1).setHardness(2.3f).setLightOpacity(16).setCreativeTab(Embers.resource_tab);
+    public static final Block BLOCK_CAMINITE_BRICK = new BlockBase(Material.ROCK, "block_caminite_brick", true).setHarvestProperties("pickaxe", 0).setHardness(1.6f).setLightOpacity(16);
+    public static final Block BLOCK_CAMINITE_BRICK_SLAB = new BlockSlabBase(BLOCK_CAMINITE_BRICK_SLAB_DOUBLE, "block_caminite_brick_slab", true).setHarvestProperties("pickaxe", 0).setIsFullCube(false).setIsOpaqueCube(false).setHardness(1.6f).setLightOpacity(1);
```

Comment:
@Klebestreifen this is the problem with the slabs I mentioned before if you want to try to figure out how to avoid this.